modules/availability.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * service_description/rhacs-cloud-service-service-description.adoc
+:_content-type: CONCEPT
+[id="availability_{context}"]
+= Availability
+
+Availability and disaster avoidance are extremely important aspects of any security platform. {rh-rhacscs-first} provides numerous protections against failures at multiple levels.
+To account for possible cloud provider failures, Red Hat established multiple availability zones.
+
+
+[id="backup-and-disaster-recovery_{context}"]
+== Backup and Disaster Recovery
+
+All {product-title-managed-short} clusters are backed up using Database backups.
+This also applies to customer data stored in the Central database.
+
+All snapshots are created using the appropriate cloud provider snapshot APIs, encrypted  and then uploaded to secure object storage, which for Amazon Web Services (AWS) is an S3 bucket.
+
+* Red Hat does not commit to a Recovery Point Objective (RPO) or Recovery Time Objective (RTO). For more information, see link:https://www.redhat.com/licenses/Appendix_4_Red_Hat_Online_Services_20221213.pdf[PRODUCT APPENDIX 4 RED HAT ONLINE SERVICES].
+
+* Site Reliability Engineering performs backups only as a precautionary measure. They are stored in the same region as the cluster.
+
+* Customers should deploy multiple availability zone Secured Clusters with workloads that follow Kubernetes best practices to ensure high availability within a region.
+
+
+[id="getting-support_{context}"]
+== Getting Support
+
+The {product-title-managed-short} includes Red Hat Standard and Premium support, which you can access using the link:https://access.redhat.com/support[Red Hat Customer Portal].
+You can open support tickets for the product "Red Hat Advanced Cluster Security Cloud Service".
+
+The Red Hat Customer Experience Engineering (CEE) responds to support tickets submitted by Limited Availability customers, while Red Hat Site Reliability Engineers (SREs) proactively monitor the health of {rh-rhacscs-first}.
```

Comment:
CEE has been renamed to XE https://source.redhat.com/departments/products_and_global_engineering/xe

I recommend avoiding internal conventions entirely. Say "Red Hat support responds...."